front/src/applications/stdcm/utils/formatStdcmConf.ts

```diff
@@ -125,32 +103,18 @@ export const checkStdcmConf = (
     );
   }
 
-  const startTime = dateTimeToIso(`${originDate}T${originTime}`);
-
-  let startDateTime: Date | null;
+  const origin = pathSteps.at(0);
+  const destination = pathSteps.at(-1);
 
-  if (stdcmV2Activated) {
-    // If stdcmV2Activated is true, we use the arrival date from either the origin or the destination
-    const firstArrival = pathSteps[0]?.arrival;
-    const lastArrival = last(pathSteps)?.arrival;
-    const isDepartureScheduled = pathSteps[0]?.arrivalType === 'preciseTime';
+  const originArrival = origin?.arrival;
+  const destinationArrival = destination?.arrival;
+  const isDepartureScheduled = origin?.arrivalType === 'preciseTime';
 
-    startDateTime = isDepartureScheduled ? new Date(firstArrival!) : new Date(lastArrival!);
-  } else if (!startTime) {
-    error = true;
-    dispatch(
-      setFailure({
-        name: t('operationalStudies/manageTrainSchedule:errorMessages.trainScheduleTitle'),
-        message: t('operationalStudies/manageTrainSchedule:errorMessages.noOriginTime'),
-      })
-    );
-    startDateTime = null;
-  } else {
-    startDateTime = new Date(startTime);
-  }
+  const startDateTime = isDepartureScheduled
+    ? new Date(originArrival!)
+    : new Date(destinationArrival!);
 
   if (
-    !isDebugMode &&
```

Comment:
why doesn’t it require debug mode now?

Reply:
We no longer need this condition because we always have a ‘searchDatetimeWindow’ defined now, which wasn't the case at the beginning.